content/applications/hr/time_off/allocations.rst

```diff
@@ -93,16 +116,16 @@ view. Both buttons open a new allocation request form.
 
 Enter the following information on the new allocation request form:
 
-- :guilabel:`Time Off Type`: select the type of time off being requested for the allocation from the
+- :guilabel:`Time Off Type`: Select the type of time off being requested for the allocation from the
   drop-down menu. After a selection is made, the title updates with the time off type.
-- :guilabel:`Validity Period`: using the calendar selector, select a start and end date the
+- :guilabel:`Validity Period`: Using the calendar selector, select a start and end date the
   allocation is valid for. By default, the current date populates the first field, and :guilabel:`No
   Limit` populates the second field. Adjust both of these dates, if desired.
```

Comment:
I seem to be having issues with this.

If I go through the 'Dashboard', and click 'New Allocation Request', from the pop-up form that appears, there is no 'Validity Period' field...

![Screenshot 2024-08-07 083427](https://github.com/user-attachments/assets/73337d4c-e935-44f6-9cb2-e3de1086c19a)

And, if I go through the 'My Allocations' page, the 'Validity Period' field *is* present, but unmodifiable (greyed-out). I understand the first date field is the current date, but there is no 'No Limit' in the second field, nor is there a way to modify it.

![Screenshot 2024-08-07 083620](https://github.com/user-attachments/assets/900ded96-4f80-40ee-bef9-efed86fa2376)

Please double-check the accuracy of this statement, and add any necessary clarification to this portion, so as to avoid any confusion, and ensure everything is being represented (and explained) effectively. Thanks.

Reply:
I was comparing these to the fields form granting allocations in the management section, and did not realize/notice they acted differently! Thank you - I have updated this so the info is correct. I explained how one field is only present when requesting allocations from one screen and that it is not modifiable.